src/sql/workbench/contrib/notebook/test/browser/notebookActions.test.ts

```diff
@@ -257,6 +258,24 @@ suite('Notebook Actions', function (): void {
 		assert.strictEqual(actualCmdId, NewNotebookAction.INTERNAL_NEW_NOTEBOOK_CMD_ID);
 	});
 
+	// test.skip('Run with Parameters Action', async function (): Promise<void> {
```

Comment:
You don't have to comment this out - just having test.skip is enough to make it not run the test.